src/Cello/mesh_Hierarchy.hpp

```diff
@@ -251,7 +251,7 @@ class Hierarchy {
   int blocking_[3];
 
   /// Periodicity of boundary conditions on faces
-  bool periodicity_[3];
+  int periodicity_[3];
```

Comment:
Out of curiosity, what is the motivation for transitioning from `bool`s to `int`s? (Are there more than 2 states or is there something else?)

Reply:
I _think_ it was because it was previously accessed as int sometimes (e.g. `get_periodicity()` arguments seem to have been `(int *) `), so I changed them all to int for consistency. Though they likely could be all changed to bool.